packages/core/admin/admin/src/content-manager/history/pages/History.tsx

```diff
@@ -166,5 +163,41 @@ const HistoryPage = () => {
   );
 };
 
-export { HistoryPage, HistoryProvider, useHistoryContext };
+/* -------------------------------------------------------------------------------------------------
+ * ProtectedHistoryPage
+ * -----------------------------------------------------------------------------------------------*/
+
+const ProtectedHistoryPage = () => {
+  const { slug } = useParams<{
+    slug: string;
+  }>();
+  const [{ query }] = useQueryParams();
+  const { permissions = [], isLoading, isError } = useSyncRbac(slug ?? '', query, 'History');
+
+  if (isLoading) {
+    return <Page.Loading />;
+  }
+
+  if ((!isLoading && isError) || !slug) {
+    return (
+      <Box height="100vh">
+        <Page.Error />
+      </Box>
```

Comment:
last thought, `EmptyStateLayoutProps` only has a finite list, should we spread props onto the `main` component instead? so you can do what you're doing here but with one less box? Or maybe it's too niche and only is required for this use case....